phy/io/kwik/creator.py

```diff
@@ -297,9 +299,18 @@ def add_clustering(self,
                                        )
 
 
-def create_kwik(prm_file=None, kwik_path=None, probe=None, **kwargs):
+def create_kwik(prm_file=None, kwik_path=None, overwrite=False,
+                probe=None, **kwargs):
     prm = _read_python(prm_file) if prm_file else {}
-    sample_rate = kwargs.get('sample_rate', prm.get('sample_rate'))
+
+    if prm:
+        assert 'spikedetekt' in prm
```

Comment:
I think that's already possible with kwik creator and add_spikes

Le mercredi 1 juillet 2015, nippoo <notifications@github.com> a écrit :

> In phy/io/kwik/creator.py
> <https://github.com/kwikteam/phy/pull/448#discussion_r33703042>:
>
> >      prm = _read_python(prm_file) if prm_file else {}
> > -    sample_rate = kwargs.get('sample_rate', prm.get('sample_rate'))
> > +
> > +    if prm:
> > +        assert 'spikedetekt' in prm
>
> todo: we should be able to convert to .kwik without running spike
> detection (i.e. separate those two processes, and manually specify / detect
> spikes using another algorithm)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy/pull/448/files#r33703042>.
>